native/cocos/platform/apple/modules/CanvasRenderingContext2DDelegate.mm

```diff
@@ -348,6 +355,9 @@ - (void)fillText:(NSString *)text x:(CGFloat)x y:(CGFloat)y maxWidth:(CGFloat)ma
     CGContextSetShouldSubpixelQuantizeFonts(_context, false);
     CGContextBeginTransparencyLayerWithRect(_context, CGRectMake(0, 0, _width, _height), nullptr);
     CGContextSetTextDrawingMode(_context, kCGTextFill);
+    if (_shadowColor && (_shadowBlur > 0 || _shadowOffsetX > 0 || _shadowOffsetY > 0)) {
```

Comment:
How about add a function to check it, such as `isNeedSetShadow`. The function name should be clear.